Cargo.toml

```diff
@@ -33,7 +33,7 @@ lightning-net-tokio = { version = "0.0.115" }
 lightning-persister = { version = "0.0.115", path = "./rust-lightning/lightning-persister" }
 lightning-rapid-gossip-sync = { version = "0.0.115" }
 magic-crypt = "3.1.12"
-miniscript = { version = "8.0", features = ["serde"] }
+miniscript = { version = "9.0.1", features = ["serde"] }
```

Comment:
thanks for spotting this, let's keep the current version though, as dependency versions are usually updated before releasing a project update and after checking that all functionality works, also in related projects

Reply:
done.